setup.cfg

```diff
@@ -7,6 +7,9 @@ max-line-length = 80
 select = C,E,F,W,B,B950
 ignore = E501,W503,E203
 
+[isort]
+profile = black
```

Comment:
[Documentation](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) says `"black"`?

Reply:
Your link is for `.isort.cfg`, while [here](https://pycqa.github.io/isort/docs/configuration/config_files.html) they don't use quotation marks for `setup.cfg`, so maybe those are slightly different dialects. Anyway, the current version seems to work...